app/src/useComprehension.js

```diff
@@ -19,42 +19,42 @@ const resultMap = new WeakMap();
 // React hook to take an array of transcript chunks and return a corresponding array of Comprehend results, one for each
 export default function useComprehension(transcriptChunks, clientParams) {
   // Dummy state variable so we can force a refresh when we receive an update
```

Comment:
This comment should be removed as it does not apply to the new state

Reply:
Will do.